components/use-case/FeatureShowcase.vue

```diff
@@ -100,6 +100,7 @@ h2 {
   }
   .img-fluid {
     width: 100%;
+    border-radius: 2%;
```

Comment:
although it's fine. We can directly use 8px here, it will make this consistent as other items

Reply:
Okay @dhaladitya108 , i thought that we don't have a particular aspect ration , that's whay I kept it 2%.